.github/workflows/npm-release-publish.yml

```diff
@@ -3,9 +3,12 @@ name: Release branch - Publish npm package
 on:
   workflow_dispatch:
     inputs:
+      release-type:
+        description: 'Release type. This must be public, beta, alpha'
+        required: true
       npm-tag:
         description: 'Npm tag for the release, e.g. latest, next, dev'
-        required: true
+        required: false
```

Comment:
This shouldn't be required for releases after GA, we may do patch versions that won't be one of these tags